pkg/autoscaler/autoscaler.go

```diff
@@ -168,7 +169,7 @@ var _ = Describe("Autoscaler should", framework.LabelAutoscaler, Serial, func()
 
 		BeforeEach(func() {
 			gatherer, err = framework.NewGatherer()
-			Expect(err).ToNot(HaveOccurred())
+			Expect(err).To(Succeed())
```

Comment:
This was correct before 😅 `Expect(err).ToNot(HaveOccurred())` is correct, it reads like a sentence